client/src/app/shared/components/markdown-text/markdown.service.ts

```diff
@@ -1,20 +1,19 @@
 import { Injectable, SecurityContext } from '@angular/core';
 import { DomSanitizer } from '@angular/platform-browser';
 
-import { IMarkdownParser, IMarkdownParserOptions } from './index';
+import { IMarkdownParser, IMarkdownParserOptions } from './markdown-parser';
```

Comment:
wouldn't importing this interfaces from `markdown-parser` would result in statically linking markdown libraries which were intended to be loaded lazily?

Reply:
Discussed, decided to make usual import

Reply:
Fixed